Portchain/auth-14-client
# Use SameSite=None for cookies

## Description

With this PR, SameSite=None is used for cookies. This was the default before recent breaking changes in Chrome 80.

The `express` dependency is bumped to a version that supports the sameSite option for `res.cookie`.

## How Has This Been Tested?

- Link a Portchain application against auth14-client with this PR
- Spin up the application locally in dev mode
- Disable the 'Cookies without SameSite must be secure' flag in Chrome
- Verify you can log in

## Screenshots:
